source/PlayState.hx

```diff
@@ -3908,6 +3908,20 @@ class PlayState extends MusicBeatState
 		}
 	}
 	#end
+	
+	public var screenGoesBlack:Bool = true;
+	function blackScreen(?screenGoesBlack:Bool = true) { // this can also be used for winter horrorland
+		if (screenGoesBlack) {
+			camGame.visible = false; //Setting both cams go invisible makes it go black? idk why lol
+			camHUD.visible = false;
+			camOther.visible = false; //why not?
```

Comment:
Why not what? If you feel the need to say something like this, I believe it would be better to explain yourself.

Reply:
I mean why not make this